classes/RackRequest.class.php

```diff
@@ -77,6 +78,7 @@ function MakeSafe(){
 		$this->LabelColor=sanitize($this->LabelColor);
 		$this->CurrentLocation=sanitize(transform($this->CurrentLocation));
 		$this->SpecialInstructions=sanitize($this->SpecialInstructions);
+		$this->RequestedAction=sanitize($this->RequestedAction);
```

Comment:
Instead of using sanitize, use a list of safe values to ensure that this is matching what data is coming in like we do with the device type.  Unless you intend on this being a freeform input field that could take anything else.

Reply:
Good point! File modified accordingly.